tests/test_manager/test_manager.py

```diff
@@ -566,27 +566,12 @@ class TestMultiAgentManagerAsyncMode(
     """Tests for MultiAgentManager in async mode."""
 
 
-class TestMultiAgentManagerAsyncModeWithPassword(
-    BaseTestMultiAgentManager
-):  # pylint: disable=unused-argument,protected-access,attribute-defined-outside-init
-    """Tests for MultiAgentManager in async mode, with password."""
-
-    PASSWORD = "password"  # nosec
```

Comment:
Why removed?

Reply:
trying to debug the problem on CI. The problem still occurs though.